src/utils/formatNumbers.ts

```diff
@@ -569,18 +581,47 @@ export function useFormatterLocales(): {
 
 // used to get the appropriate formatter functions with the correct locales passed into them
 export function useFormatter() {
+  const activeLocalCurrency = useActiveLocalCurrency()
   const { formatterLocale, formatterLocalCurrency } = useFormatterLocales()
 
+  const activeLocalCurrencyIsUSD = activeLocalCurrency === GqlCurrency.Usd
+  const { data: localCurrencyConversionRate, isLoading: localCurrencyConversionRateIsLoading } =
+    useLocalCurrencyConversionRate(activeLocalCurrency, activeLocalCurrencyIsUSD)
+
+  const previousSelectedCurrency = usePrevious(activeLocalCurrency)
+  const previousConversionRate = usePrevious(localCurrencyConversionRate)
+
+  const shouldFallbackToPrevious = !localCurrencyConversionRate && localCurrencyConversionRateIsLoading
```

Comment:
can we show a loading state instead of the previous conversion rate? or also show the previous symbol as well? For large diffs such as USD -> JPY the symbol changes and you're momentarily flashed a very innaccurate price. This would be further emphasized on slower internet connections

Reply:
I think this is a weird caching thing by vercel because thats how it previously worked so i updated it so it stayed on the previous symbol and currency so there was no flashing and its smooth - you can observe this if you run it locally